ripd/ripd.c

```diff
@@ -282,8 +282,14 @@ struct rip_info *rip_ecmp_replace(struct rip *rip, struct rip_info *rinfo_new)
  */
 struct rip_info *rip_ecmp_delete(struct rip *rip, struct rip_info *rinfo)
 {
-	struct route_node *rp = rinfo->rp;
-	struct list *list = (struct list *)rp->info;
+	struct route_node *rp;
+	struct list *list;
+
+	if (rinfo == NULL)
```

Comment:
Technically a code change to add the NULL check?

Don't want to make Coverity angry by implying things could be NULL when they can't